src/motile_plugin/import_export/load_tracks.py

```diff
@@ -0,0 +1,96 @@
+from csv import DictReader
+
+import networkx as nx
+import numpy as np
+from motile_toolbox.candidate_graph import NodeAttr
+
+from motile_plugin.data_model import SolutionTracks
+
+
+def tracks_from_csv(
+    csvfile: str,
+    selected_columns: dict,
+    extra_columns: dict,
+    segmentation: np.ndarray | None = None,
+    scale: list[float] | None = None,
+) -> SolutionTracks:
+    """Assumes a csv similar to that created from "export tracks to csv" with columns:
+        t,[z],y,x,id,parent_id,[seg_id]
+    Cells without a parent_id will have an empty string or a -1 for the parent_id.
+
+    Args:
+        csvfile (str):
+            path to the csv to load
+        selected_columns (dict): a dictionary mapping the attributes "t", "z (optional)", "y", "x", "node_id", "seg_id (if seg provided)", "parent_id" to columns of the csv file
+        extra_columns (dict): a dictionary mapping optional additonal attributes to columns of the csv file
+        segmentation (np.ndarray | None, optional):
+            An optional accompanying segmentation.
+            If provided, assumes that the seg_id column in the csv file exists and
+            corresponds to the label ids in the segmentation array
+
+    Returns:
+        Tracks: a tracks object ready to be visualized with
```

Comment:
I think we should delete view_external_tracks and just add_tracks to the tracks list

Reply:
Done! I also updated the documentation and sample script.